desc/geometry/surface.py

```diff
@@ -783,6 +783,43 @@ def fun_jax(zeta_hat, theta, zeta):
         else:
             return offset_surface
 
+    def get_axis(self):
+        """Get the axis of the surface.
```

Comment:
should add some explanation that this is the geometric centroid (and define how its computed), not a magnetic axis.